vms/secp256k1fx/camino_fx.go

```diff
@@ -268,3 +257,39 @@ func (fx *Fx) verifyMultisigUnorderedCredentials(tx UnsignedTx, creds []verify.V
 
 	return nil
 }
+
+// SplitPrivateKeys splits an array of PrivateKeys into `from` and `signers`
+// The delimiter is a `nil` PrivateKey.
+// If no delimiter exists, the given PrivateKeys are used for both from and signing
+// Having different sets of `from` and `signer` allow multisignatuew alias feature
+func SplitPrivateKeys(keys []*crypto.PrivateKeySECP256K1R) (set.Set[ids.ShortID], []*crypto.PrivateKeySECP256K1R) {
+	// find nil key which splits froms and signer
+	splitIndex := len(keys)
+	for index, key := range keys {
+		if key == nil {
+			splitIndex = index
+			break
+		}
+	}
+
+	if splitIndex == len(keys) {
```

Comment:
We can remove this entirerly and make just 2 loops as below but both indexes on `keys` slice:

```go
	for index := 0; index < splitIndex; index++ {
		from.Add(keys[index].PublicKey().Address())
	}
	
	//...
	for index := splitIndex+1; index < len(keys); index++ {
		signer = append(signer, keys[index])
	}
```

Reply:
But only with an innr branch (if / else), and I don't like it -> stays like it is

Reply:
Oh, I see, the return values diff. 👍